hedera-node/src/test/java/com/hedera/services/state/expiry/renewal/RenewalHelperTest.java

```diff
@@ -336,7 +337,6 @@ void removesAutoAccountEntityWhenExpired() {
 		subject.classify(brokeExpiredAccountNum, now);
 		subject.removeLastClassifiedAccount();
 
-		verify(backingAccounts).remove(expiredKey.toGrpcAccountId());
```

Comment:
we should also assert that the expiredKey-Id is removed from the accountsMap.

Reply:
Added. Thanks @anighanta